funnel/models/contact_exchange.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+from . import db, BaseMixin
+
+__all__ = ['ContactExchange']
+
+
+class ContactExchange(BaseMixin, db.Model):
+    __tablename__ = 'contact_exchange'
+    user_id = db.Column(None, db.ForeignKey('user.id'))
+    proposal_space_id = db.Column(None, db.ForeignKey('proposal_space.id'))
+    participant_id = db.Column(None, db.ForeignKey('participant.id'))
+    __table_args__ = (db.UniqueConstraint('user_id', 'proposal_space_id', 'participant_id'),)
```

Comment:
Since there's nothing else in this table, this could be a three column primary key with no native `id` column.